packages/engine/src/lib/helper/trigger-helper.ts

```diff
@@ -50,6 +52,7 @@ export const triggerHelper = {
                 prefix,
                 flowId: params.flowVersion.flowId,
                 workerToken: params.workerToken,
+                runId: get(params, 'flowRunId', ''),
```

Comment:
remove `lodash` and the console log above.

replace this with constnats.flowRunId

Reply:
Ah, okay. So a trigger won't have the run-scope at all?